meet_balls_app/tests.py

```diff
@@ -195,3 +196,83 @@ def test_login_page_navigation(self, client):
         response = client.get(login_url)
         assert response.status_code == 200
         assert response.request['PATH_INFO'] == '/login/'
+
+
+@pytest.mark.django_db
+class TestProfile:
+
+    def test_profile_view_no_data(self, client, player):
+        url = reverse('profile', kwargs={'id': player.user.id})
+        client.force_login(player.user)
+        response = client.get(url)
+        assert response.status_code == 200
+        assert player.user.username in response.content.decode()
+        assert player.favorite_ball_game in response.content.decode()
+
+    def test_profile_view_with_data(self,
+                                    client,
+                                    player,
+                                    player_rating,
+                                    court,
+                                    court_ball_game,
+                                    game_event,
+                                    game_event_player,
+                                    ):
+
+        url = reverse('profile', kwargs={'id': player.user.id})
+        client.force_login(player.user)
+        response = client.get(url)
+        assert response.status_code == 200
+        assert player.user.username in response.content.decode()
+        assert player.favorite_ball_game in response.content.decode()
+        assert str(player_rating.rating) in response.content.decode()
```

Comment:
Are we asserting that the number 5 is somewhere within the profile? I think this is a bit of a hit-or-miss here? If you'd do the same with the numbers 0-9, how many of those will pass? I think we need to assert on something more convincing (this applies to a lesser extent on the ball game as well).